### PR TITLE
[JENKINS-67713] Align Polling Log "View as plain text" link with sidepanel

### PR DESCRIPTION
**Pull request by Langer0416**
See [JENKINS-67713](https://issues.jenkins.io/browse/JENKINS-67713).

### Proposed changelog entries

* Move Polling Log "View as plain text" link to side panel.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/hudson/triggers/SCMTrigger/BuildAction/index.jelly
Comment (NotMyFault) [inline core/src/main/resources/hudson/triggers/SCMTrigger/BuildAction/index.jelly:34]: That looks quite out of place


I'd vouch for removing it from the rightspace at all and integrating it into the sidebar, like the "Console Output" -> "View as plain text" way e.g.:

**Comment by Langer0416** [comment]
I had followed your suggestions to make some modifications, have a look at it.
image

**Review by timja — APPROVED** [review]
untested. code looks good and screenshot looks much better than before

**Review by NotMyFault — APPROVED** [review]

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
